cmd/snap/cmd_auto_import.go

```diff
@@ -165,5 +174,19 @@ func (x *cmdAutoImport) Execute(args []string) error {
 		defer doUmount(mp)
 	}
 
-	return autoImportFromAllMounts()
+	added, err := autoImportFromAllMounts()
+	if err != nil {
+		return err
+	}
+
+	isOwned, err := isOwned()
```

Comment:
We don't need to check this here.. seems fine to just try to create, and the server will ensure this is appropriate.